airflow/example_dags/example_python_operator.py

```diff
@@ -135,4 +141,16 @@ def callable_external_python():
         external_python_task = callable_external_python()
         # [END howto_operator_external_python]
 
-        run_this >> external_python_task
+        external_classic = ExternalPythonOperator(
+            task_id="external_python_classic",
+            python=os.fspath(sys.executable),
```

Comment:
```suggestion
            python=sys.executable,
```

But I feel it’s probably more useful to provide a more practical example, e.g.

```suggestion
            python="/opt/Python-3.11.0/bin/python",
```

Reply:
It does not make it runnable (primary purpose of this is to be able to run example_dag in airflow started locally with examples. But possibly extracting it to properly named variable migt make more sense.

Reply:
I reworked it a bit, also added more detailed description in the HowTo about the binary path and changed it name to `PATH_TO_PYTHON_BINARY` -  and added classic way of using both external and virtualenv operator as examples.